code/modules/clothing/shoes/clownshoes.dm

```diff
@@ -0,0 +1,204 @@
+	. = ..()
+	AddComponent(/datum/component/squeak, list('sound/effects/footstep/catwalk3.ogg'=1,'sound/effects/footstep/carpet5.ogg'=1), 100)
+
+
+/obj/item/clothing/shoes/clown_shoes/hydro
+	desc = "A bit prickly, and somewhat loose. Perhaps you can get a new sole? Ctrl-click to toggle waddle dampeners."
+	name = "clown grass shoes"
+	icon_state = "clown_hydro"
+	inhand_icon_state = "clown_shoes_hydro"
+
+/obj/item/clothing/shoes/clown_shoes/hydro/Initialize()
+	. = ..()
+	AddComponent(/datum/component/squeak, list('sound/effects/footstep/grass2.ogg'=1,'sound/effects/footstep/grass3.ogg'=1), 100)
+
+
+/obj/item/clothing/shoes/clown_shoes/clogs
+	desc = "Great sole support, but shouldn't these bee yellow? Ctrl-click to toggle waddle dampeners."
+	name = "clown clogs"
+	icon_state = "clown_clogs"
+	inhand_icon_state = "clown_shoes_clogs"
+
+/obj/item/clothing/shoes/clown_shoes/clogs/attackby(obj/H, mob/user, loc)
+	..()
+	if (H.type == /obj/item/reagent_containers/honeycomb || H.type== /obj/item/grown/sunflower)
+		var/location = get_turf(user)
+		new /obj/item/clothing/shoes/clown_shoes/dutch(location)
+		qdel(H)
+		qdel(src)
+	return
+
+/obj/item/clothing/shoes/clown_shoes/clogs/Initialize()
+	. = ..()
+	AddComponent(/datum/component/squeak, list('sound/effects/footstep/woodclaw1.ogg'=1,'sound/effects/footstep/woodclaw3.ogg'=1), 100)
+
+
+/obj/item/clothing/shoes/clown_shoes/dutch
+	desc = "Why are dutch clowns not tall? All the big ones bounced out the ceiling! TURN ON YOUR WADDLE DAMPENERS WITH CTRL-CLICK FOR GODS SAKE!!"
+	name = "dutch clown clogs"
+	icon_state = "clown_dutch"
+	inhand_icon_state = "clown_shoes_dutch"
+
+/obj/item/clothing/shoes/clown_shoes/dutch/Initialize()
+	. = ..()
+	AddComponent(/datum/component/squeak, list('sound/effects/clownstepdutch.ogg'=1), 100)
+
+
+/obj/item/clothing/shoes/clown_shoes/med
+	desc = "A more sanitary version of the clown shoes.  Ctrl-click to toggle waddle dampeners."
+	name = "medical clown crocks"
+	icon_state = "clown_med"
+	inhand_icon_state = "clown_shoes_med"
+	permeability_coefficient = 0.01
+
+/obj/item/clothing/shoes/clown_shoes/med/Initialize()
+	. = ..()
+	AddComponent(/datum/component/squeak, list('sound/effects/clownstepcrocks.ogg'=1), 50)
+
+
+/obj/item/clothing/shoes/clown_shoes/janny
+	desc = "Clown shoes for when the joke has gone wrong.  Ctrl-click to toggle waddle dampeners."
+	name = "clown rubberboots"
+	icon_state = "clown_janny"
+	inhand_icon_state = "clown_shoes_janny"
+	permeability_coefficient = 0.01
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 40, "acid" = 75)
+	can_be_bloody = FALSE
+
+/obj/item/clothing/shoes/clown_shoes/janny/Initialize()
+	. = ..()
+	AddComponent(/datum/component/squeak, list('sound/effects/clownstepthicc.ogg'=1), 50)
+
+/obj/item/clothing/shoes/clown_shoes/galoshes
+	desc = "Clown shoes for when the joke has gone terribly wrong.  Ctrl-click to toggle waddle dampeners."
```

Comment:
```suggestion
	desc = "Clown shoes for when the joke has gone terribly wrong. This particular pair is designed to prevent slipping on wet surfaces."
```